fuzzers/074-dump_all/ignored_wires/artix7/xc7a100tfgg676-1_ignored_wires.txt

```diff
@@ -0,0 +1,132 @@
+LIOI3_TBYTETERM_X0Y113/LIOI_I2GCLK_BOT1
```

Comment:
I think that the ignored wires should be sorted

Reply:
I agree!   I made a lot of use of 'sort', 'sed', and 'grep' comparing the ignored wires against those for the 50T.

Reply:
I have sorted the wires, just using the basic alphabetical sort.  This matches what was done for the 200T part.